app/piping/plumber.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+
+class Plumber(object):
+    def __init__(self, context):
+        self._context = context
+
+    def plumb_item(self, item):
+        interesting_properties = self.get_properties_of_interest(item)
```

Comment:
Not sure what interesting_properties means, can you change the name to something more descriptive